src/super_gradients/training/sg_model/sg_model.py

```diff
@@ -1719,6 +1734,21 @@ def evaluate(self, data_loader: torch.utils.data.DataLoader, metrics: MetricColl
 
         if self.multi_gpu == MultiGPUMode.DISTRIBUTED_DATA_PARALLEL:
             logging_values = reduce_results_tuple_for_ddp(logging_values, next(self.net.parameters()).device)
+
+        pbar_message_dict = get_train_loop_description_dict(logging_values,
+                                                            metrics,
+                                                            self.loss_logging_items_names)
+
+        self.valid_monitored_values = sg_model_utils.update_monitored_values_dict(
+            monitored_values_dict=self.valid_monitored_values, new_values_dict=pbar_message_dict)
+
+        if not silent_mode:
+            progress_bar_data_loader.write("===========================================================")
+            sg_model_utils.display_epoch_summary(epoch=context.epoch, n_digits=4,
```

Comment:
You are calling "display epoch summary" inside our evaluate function, which is used both by validate, and test. For the rather, it does not make sense. You need to make sure such prints are not done when calling SgModel.test